### PR TITLE
docs: Create README.md

### LINKED ISSUE #2: This project needs a README
We should add a README to this repository to track contributions and to leave documentation behind for contributors and maintainers.

### PR DESCRIPTION
**Pull request by mcataford**
Closes #2

### CODE REVIEW

**Comment by mcataford** [comment]
@all-contributors please add @iamogbz for code and ideas

File: README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Try it out connoisseur
+
+```sh
+brew install tophat/bar/yvm
+```
+
+| Formulae         | Sip                  | Brewery            |
+| ---------------- | -------------------- | ------------------ |
+| `tophat/bar/yvm` | Yarn Version Manager | https://yvm.js.org |
```
Comment (iamogbz) [inline README.md]: Once #3 is workable

Comment (mcataford) [reply README.md]: 👍

**Comment by mcataford** [comment]
@all-contributors please add @iamogbz for code and ideas

**Comment by mcataford** [comment]
@all-contributors please add @mcataford for doc

**Comment by mcataford** [comment]
@all-contributors please add @mcataford for doc

**Pull request merged**